packages/hdwallet-provider/package.json

```diff
@@ -37,7 +37,8 @@
     "ganache-core": "^2.8.0",
     "mocha": "6.2.0",
     "ts-node": "^8.4.1",
-    "typescript": "^3.6.3"
+    "typescript": "^3.6.3",
+    "web3": "1.2.2"
```

Comment:
Does this really need to go to `devDependencies`?!

Reply:
Yup, I think the original import was incorrect and pointing to `@types/web3` instead of `web3`.

Reply:
oh, nm. missed the other import!